### DISCUSSION TITLE
v0.6 - STM32 and RP2040, Logger, Error Corrections

### DISCUSSION THREAD

**Discussion by pschatzmann** [pschatzmann/arduino-vs1053#5] [Announcements]
- I added a logger which support log levels and improved the error handling a bit.
- Finally I could confirm that the functionality is also working with
  - RP2040
  - STM32

<hr /><em>This discussion was created from the release <a href='https://github.com/pschatzmann/arduino-vs1053/releases/tag/v0.6'>v0.6 - STM32 and RP2040, Logger, Error Corrections</a>.</em>

**Comment by luke2023** [comment]
Thank you so much!I can't believe that  you dealt with so much code yourself!
